QPM/Providers/RemoteQPMDependencyResolver.cs

```diff
@@ -194,16 +197,24 @@ private void CopyTo(string downloadFolder, in Config myConfig, in SharedConfig s
         {
             var baseDst = Path.Combine(myConfig.DependenciesDir, sharedConfig.Config.Info.Id);
             var dst = Path.Combine(myConfig.DependenciesDir, sharedConfig.Config.Info.Id, sharedConfig.Config.SharedDir);
+            var dstExpanded = Path.GetFullPath(dst);
             var root = Utils.GetSubdir(downloadFolder);
             var src = Path.Combine(root, sharedConfig.Config.SharedDir);
             if (!Directory.Exists(dst) || !Utils.FolderHash(src).SequenceEqual(Utils.FolderHash(dst)))
             {
                 if (Directory.Exists(dst))
                     Utils.DeleteDirectory(dst);
                 Console.WriteLine($"Copying: {src} to: {dst}");
-                Utils.CopyDirectory(src, dst);
+
+                // Create parent directories
```

Comment:
Can you don't and just call create dir on `baseDst`

Reply:
Why do you still do this? Creating and deleting just for the sake of creating the base dirs is literally harder than creating the base dirs themselves.

Reply:
Or you could handle the creation of subdirs in your `SymLinkOrCopyDirectory` function